### PR TITLE
Switch kernel to use ortoolkit and plumb through choice of algorithm

### PR DESCRIPTION
**Pull request by jnation3406**
I might keep making some minor tweaks, but it is working and basically done.

### CODE REVIEW

File: test/test_fullscheduler_ortoolkit.py [hunk 1]
```diff
@@ -1,109 +1,44 @@
 #!/usr/bin/env python
```
Comment (eheinrich) [inline test/test_fullscheduler_ortoolkit.py:1]: Is this only testing cbc? If so, this file might be better named `test_fullscheduler_cbc.py`, in line with the names of the other test files that test gurobi and glpk.

**Review by eheinrich — COMMENTED** [review]
The files that were removed - fullscheduler_{v5,v6,v7}.py - were those just historical?

I think it might be a good idea to update the README. There's a section in there that says we're planning to use OR-tools, which should probably be replaced.

File: adaptive_scheduler/kernel/fullscheduler_ortoolkit.py [hunk 2]
```diff
@@ -96,101 +109,100 @@ def schedule_all(self, timelimit=0):
...
 
-        # set the objective, and maximize it
-        m.setObjective(objective)
-        m.modelSense = GRB.MAXIMIZE
-
-        # impose a time limit on the solve
+        # impose a time limit (ms) on the solve
         if timelimit > 0:
-            m.params.timeLimit = timelimit
+            solver.SetTimeLimit((int)(timelimit * 1000))
```
[the remark this answers is not in the document]
Comment (jnation3406) [reply adaptive_scheduler/kernel/fullscheduler_ortoolkit.py:182]: oh yeah probably... i must have been thinking in a different language so I'll change it to be more pythonic

File: test/test_fullscheduler_ortoolkit.py (hunk 1 above)
Comment (jnation3406) [reply test/test_fullscheduler_ortoolkit.py:1]: yup

**Comment by jnation3406** [comment]
the fullscheduler_v5/6/7 were other kernels that could be used instead of gurobi. v6/7 used versions of glpk, which is now included as an option through ortools. v5 used scipy, but I don't think it was very effective, so its "replaced" with cbc as an option now.

**Review by eheinrich — APPROVED** [review]

**Pull request merged**